config/locales/client.en.yml

```diff
@@ -2445,6 +2445,10 @@ en:
       no_results: "No results"
       search: "Search…"
       label: "Select options"
+    d_otp:
+      screen_reader:
+        one: Fill one char
```

Comment:
This seems not user-friendly. Since OTPs are mostly numbers, should we say "Enter a number" or something like that?

Reply:
Yes sounds good :+1: Even though as I said below, it's not something you decide yourself like a password, you are given something to fill it, so you know what it is.